tests/.eslintrc.js

```diff
@@ -2,5 +2,6 @@ module.exports = {
   extends: 'plugin:shopify/esnext',
   rules: {
     'import/no-extraneous-dependencies': 'off',
+    'prettier/prettier': 'off',
```

Comment:
Alternatively, we could extend the `plugin:shopify/prettier` config and run prettier against the `/tests` code.

Reply:
I like the alternative better